Error: Cannot find module 'vscode-languageserver/lib/utils/is'

Hello! I seem to be struggling to run the executable - after installing `spectral-language-server` and running the `./node_modules/.bin/spectral-language-server` I receive this error:

```
/Users/williamboman/.local/share/nvim/lsp_servers/spectral/node_modules/.bin/spectral-language-server"	"stderr"	"node:internal/modules/cjs/loader:930
  throw err;
  ^

Error: Cannot find module 'vscode-languageserver/lib/utils/is'
Require stack:
- /Users/williamboman/.local/share/nvim/lsp_servers/spectral/node_modules/spectral-language-server/dist/spectral-language-server/server.js
- /Users/williamboman/.local/share/nvim/lsp_servers/spectral/node_modules/spectral-language-server/bin/spectral-language-server
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:927:15)
    at Function.Module._load (node:internal/modules/cjs/loader:772:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/Users/williamboman/.local/share/nvim/lsp_servers/spectral/node_modules/spectral-language-server/dist/spectral-language-server/server.js:7:14)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1124:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:816:12)
    at Module.require (node:internal/modules/cjs/loader:999:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/williamboman/.local/share/nvim/lsp_servers/spectral/node_modules/spectral-language-server/dist/spectral-language-server/server.js',
    '/Users/williamboman/.local/share/nvim/lsp_servers/spectral/node_modules/spectral-language-server/bin/spectral-language-server'
  ]
}
```

--- a/script/vscode-spectral-build.sh
+++ b/script/vscode-spectral-build.sh
@@ -8,7 +8,7 @@ mkdir -p $DIR/../dist
 
 # clone
 cd $DIR/../tmp
-git clone --depth=1 git@github.com:stoplightio/vscode-spectral.git vscode-spectral
+git clone --depth=1 git@github.com:stoplightio/vscode-spectral.git -b v0.2.5 vscode-spectral
 
 # pull
 cd $DIR/../tmp/vscode-spectral

